service/job.go

```diff
@@ -83,6 +83,19 @@ func (s *CaptionsService) GetJob(r *http.Request) (int, interface{}, error) {
 	return http.StatusOK, job, nil
 }
 
+// CancelJob cancels a given Job by its ID
+func (s *CaptionsService) CancelJob(r *http.Request) (int, interface{}, error) {
+	id := web.Vars(r)["id"]
+	canceled, err := s.client.CancelJob(id)
+	if err != nil {
+		return http.StatusNotFound, nil, captionsError{err.Error()}
```

Comment:
This `err` might be from `c.DB.UpdateJob(jobID, job)`, in which case returning `404` could be confusing.